packages/storybook-a11y-test/src/index.ts

```diff
@@ -28,32 +28,167 @@ declare global {
 
 const FORMATTING_SPACER = '    ';
 
-function getBrowser() {
-  return puppeteer.launch({
-    args: ['--no-sandbox', '--disable-setuid-sandbox'],
-  });
-}
+export class A11yTests {
+  #browser: Browser | undefined;
+  #iframePage: Page | undefined;
```

Comment:
Yeah but this will fail if used with something that is not the iframe since for the test pages we expect that path + id. Not too confident on this name as well.